test/EndToEnd/GivenDotNetUsesMSBuild.cs

```diff
@@ -19,7 +19,7 @@ public void ItCanNewRestoreBuildRunCleanMSBuildProject()
             {
                 string projectDirectory = directory.Path;
 
-                string newArgs = "console --debug:ephemeral-hive";
+                string newArgs = "console -f netcoreapp1.1 --debug:ephemeral-hive";
```

Comment:
Should the default here become 1.1? @mlorbetske does the latest template update make that change?